.circleci/config.yml

```diff
@@ -28,6 +28,11 @@ jobs:
                 opam pin add -yn "${f%.opam}.${version}.dev" .
               fi
             done
+      - run:
+          name: Synchronize with opam-repository
+          command: |
+            opam repo add upstream git+https://github.com/ocaml/opam-repository.git
+            opam update
```

Comment:
Just a side note that this patch is not needed anymore since `tls-miou-unix.1.0.3` is available from the `opam.ocaml.org` cache. You can delete it :+1:.